com.unity.render-pipelines.universal/CHANGELOG.md

```diff
@@ -69,7 +69,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - The 2D Renderer now supports camera stacking.
 - 2D shaders now use half-precision floats whenever precise results are not necessary.
 - Removed the ETC1_EXTERNAL_ALPHA variant from Shader Graph Sprite shaders.
-- Eliminated GC allocations caused by the 2D Renderer.
```

Comment:
Is this change intentional? Seems like it's overriding another entry in the log.

Reply:
Yes. It was a duplicate entry.